test/stdlib/Accelerate.swift

```diff
@@ -10,63 +10,267 @@ import Accelerate
+        let _ = BNNSLayerData(data: nil, data_type: .indexed8)
+    }
+    
+    AccelerateTests.test("BNNS/Activation") {
+        expectEqual(BNNSActivation.identity.function, .identity)
+        let id = BNNSActivation(function: .identity)
+        expectTrue(id.alpha.isNaN)
+        expectTrue(id.beta.isNaN)
+    }
+    
+}
+
+if #available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *) {
+    
+    AccelerateTests.test("Quadrature/QNG") {
+        var diameter: Double = 25
+        
+        // New API
+        let quadrature = Quadrature(integrator: .qng,
+                                    absoluteTolerance: 1.0e-8,
+                                    relativeTolerance: 1.0e-2)
+        
+        let result = quadrature.integrate(over: 0.0 ... diameter) { x in
+            let radius = diameter * 0.5
+            return sqrt(radius * radius - pow(x - radius, 2))
+        }
+        
+        // Legacy API
+        var integrateFunction: quadrature_integrate_function = {
+            return quadrature_integrate_function(
+                fun: { (arg: UnsafeMutableRawPointer?,
+                    n: Int,
+                    x: UnsafePointer<Double>,
+                    y: UnsafeMutablePointer<Double>
+                    ) in
+                    
+                    guard let diameter = arg?.load(as: Double.self) else {
+                        return
+                    }
+                    
+                    let r = diameter * 0.5
+                    
+                    (0 ..< n).forEach { i in
+                        y[i] = sqrt(r * r - pow(x[i] - r, 2))
+                    }
+            },
+                fun_arg: &diameter)
+        }()
+        
+        var options = quadrature_integrate_options(integrator: QUADRATURE_INTEGRATE_QNG,
+                                                   abs_tolerance: 1.0e-8,
+                                                   rel_tolerance: 1.0e-2,
+                                                   qag_points_per_interval: 0,
+                                                   max_intervals: 0)
+        
+        var status = QUADRATURE_SUCCESS
+        var legacyEstimatedAbsoluteError: Double = 0
+        
+        let legacyResult = quadrature_integrate(&integrateFunction,
+                                                0.0,
+                                                diameter,
+                                                &options,
+                                                &status,
+                                                &legacyEstimatedAbsoluteError,
+                                                0,
+                                                nil)
+        
+        switch result {
+        case .success(let integralResult, let estimatedAbsoluteError):
+            expectEqual(integralResult, legacyResult)
+            expectEqual(estimatedAbsoluteError, legacyEstimatedAbsoluteError)
+        case .failure( _):
+            expectTrue(false)
```

Comment:
There is `expectationFailure()` function you can use.
```suggestion
            expectationFailure()
```

Reply:
fixed (pending push)